tests/unit/stream_alert_alert_processor/test_outputs/test_pagerduty.py

```diff
@@ -599,8 +600,6 @@ def test_dispatch_success_bad_policy(self, get_mock, post_mock, log_mock):
                                               rule_name='rule_name',
                                               alert=get_alert(context=ctx)))
 
-        log_mock.assert_called_with('Successfully sent alert to %s', self.SERVICE)
```

Comment:
I think you should be able to leave this test still - can you confirm?

Reply:
Yes, just confirmed it. Thanks!